static/js/Story.jsx

```diff
@@ -127,8 +127,93 @@ StorySheetList.propTypes = {
     toggleSignUpModal: PropTypes.func
 };
 
+function capitalizeWords(str) {
+  const words = str.split(' ');
+
+  const capitalizedWords = words.map(word => {
+    return word.charAt(0).toUpperCase() + word.slice(1);
+  });
+  const result = capitalizedWords.join(' ');
+
+  return result;
+}
+const reviewStateToClassNameMap = {
+    "reviewed": "reviewed",
+    "not reviewed": "notReviewed",
+    "edited": "edited"
+}
+const reviewStateToDisplayedTextMap = {
+    "reviewed": "Reviewed",
+    "not reviewed": "Not Reviewed",
+    "edited": "Edited"
+}
+const ReviewStateIndicator = ({reviewState, callBack}) => {
+    if (!reviewState){
+        return null}
+    let reviewStateClassName = reviewStateToClassNameMap[reviewState];
+    let displayedText = reviewStateToDisplayedTextMap[reviewState];
+    return (
+    <div className={`button extraSmall reviewState ${reviewStateClassName}`} onClick={callBack}>
+        {displayedText}
+    </div>)
+
+};
+
+const PromptWrapper = ({text}) => {
+    let promptComponent = null;
+    if (Sefaria.interfaceLang == "english"){
```

Comment:
There is a lot of duplicated code here between English and Hebrew cases. I believe it should be possible to remove all duplication.